src/site/_ru/_betterbook.yaml

```diff
@@ -1,5 +1,5 @@
 toc:
-  - title: "Web <strong>Fundamentals</strong>"
+  - title: "Основополагающие <strong>веб-стандарты</strong>"
```

Comment:
I believe the emphasis is wrong here.  The brand guidelines we have are for bolding "Fundamentals"